### ISSUE TITLE
3DS challenge not appearing on Android; appears on iOS with the same card

### ISSUE THREAD

**Issue by ptrbrynt** [pikaju/flutter-braintree#133]
I'm calling the drop-in UI with `requestThreeDSecureVerification: true`.

On iOS, the 3DS flow is launched and completes as expected. On Android, the Drop-in UI closes after payment method selection and the payment then fails; I assume this is because no 3DS verification took place.

Curious why this behavior is different across platforms. We're testing the same payment using the same card on each platform.

Thanks in advance for your support.

**Comment by nicolobozzato** [comment]
Did you add the intent filter for BraintreeBrowserSwitchActivity in the manifest? Did you also valorize field amount in the BraintreeDropInRequest?

**Comment by ptrbrynt** [comment]
Yes - `BraintreeBrowserSwitchActivity` is configured correctly. Not sure what you mean by "valorize"? We are passing an `amount` value.
